_partials/self-hosted/_scar-migration.mdx

```diff
@@ -0,0 +1,354 @@
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}" \
+        --data-raw "$(scar_request)")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Scar Repo"
+    else
+        echo "Success Setting the Scar Repo. Waiting for a minute to get the changes syncing"
+        sleep 60
+    fi
+    }
+
+    update_instance_type() {
+    response=$(curl -k --write-out '%{http_code}' --location "${ENDPOINT}/v1/jobs/internal/syncCloudInstanceTypesData/run" \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Instance Type"
+    else
+        echo "Started Syncing Storage Type from the palette Service"
+    fi
+    }
+
+    update_storage_type() {
+    response=$(curl -k --write-out '%{http_code}' --location "${ENDPOINT}/v1/jobs/internal/syncCloudStorageTypesData/run" \
+        --header 'Content-Type: application/json' \
+        --header 'Accept: application/json' \
+        --header "Cookie: Authorization=${authtoken}")
+    if [[ "${response}" != "204" ]]; then
+        echo "Error: Unable to update the Storage Type"
+    else
+        echo "Started Syncing Storage Type from the palette Service"
+    fi
+    }
+
+    get_auth_token
+    update_scar_repo
+    update_instance_type
+    update_storage_type
+
+    echo "Wait for 5 minute for the sync process to complete."
+    EOF1
+    ```
+
+10. Grant execution permissions to the `scar-registry-update.sh` script.
+
+    ```shell
+    chmod +x scar-registry-update.sh
+    ```
+
+11. Invoke the script to update the SCAR endpoint. Replace `<cluster-url>` with the address of your {props.edition} instance and
+    `<sys-admin-password>` with the system administrator password.
+
+    ```shell
+    ./scar-registry-update.sh <cluster-url> admin <sys-admin-password>
+    ```
+
+    Consider the following example for reference.
+
+    ```shell hideClipboard
+    ./scar-registry-update.sh https://example.spectrocloud.com admin examplepassword
+    ```
+
+    The following message confirms that the script has completed its tasks.
+
+    ```text hideClipboard
+    Wait for 5 minutes for the sync process to complete.
+    ```
+
+## Validate
+
+Use the following steps to verify that the endpoint was updated successfully.
+
+1. Export your {props.edition} credentials as environment variables. Replace `<cluster-url>` with the
+address of your {props.edition} cluster and `<sys-admin-password>` with the system administrator password.
+
+    ```shell
+    export ENDPOINT=<cluster-url>
+    export PASSWORD=<sys-admin-password>
+    ```
+
+2. Issue the following command to confirm that the endpoint was correctly updated.
```

Comment:
Break this step up. [Here](https://docs.spectrocloud.com/enterprise-version/system-management/change-cloud-config/) is an example of how we have done it in the past.

Reply:
:white_check_mark: